lib/libutee/default_bold.c

```diff
@@ -0,0 +1,1009 @@
+/*
```

Comment:
What are the "additional packages" you mentioned? If it's reasonably easy to get them (like they come by default with recent distribs or you just need one apt-get), I'd prefer not to commit the generated file. Now, if it's more complex, then OK not a big deal. But Pascal has the final say anyway ;)